worker/main.tf

```diff
@@ -91,7 +91,6 @@ resource "aws_launch_configuration" "launch_config" {
     volume_size           = "${var.root_storage_size}"
     volume_type           = "${var.root_storage_type}"
     delete_on_termination = true
-    encrypted             = "${var.storage_encrypted_at_rest}"
```

Comment:
@gozer did you intend to remove this?

Reply:
Yes, root volumes don't support encryption at run-time, it's a proprety of the initial AMI they are based from.